src/snowflake/cli/plugins/snowpark/package_utils.py

```diff
@@ -192,6 +194,82 @@ def download_unavailable_packages(
         )
 
 
+def pip_wheel(
+    requirements_file: Optional[str],
+    package_name: Optional[str],
+    download_dir: Path,
+    index_url: Optional[str],
+    dependencies: bool = True,
+):
+    command = ["-m", "pip", "wheel", "-w", str(download_dir)]
+    if package_name:
+        command.append(package_name)
+    if requirements_file:
+        command += ["-r", str(requirements_file)]
+    if index_url:
+        command += ["-i", index_url]
+    if not dependencies:
+        command += ["--no-deps"]
```

Comment:
Could you decide on one 😃 `+=` or `append`?

Reply:
Append is the proper way

Reply:
Done